docs/system-settings/key-profiles.md

```diff
@@ -66,30 +66,33 @@ the fader to 0.
 ### Groups
 The blue, grey or touch key can be allocated to **Disabled**,
 **Select Group**, **Flash Fixtures**, **Timed Flash**, **Flash Master**, **Timed Flash
-Master**, **Swop Fixtures**.
+Master**, **Swop Fixtures** - see [Fixture Groups](../controlling-fixtures/fixture-groups)
```

Comment:
This needs to be a link to the MarkDown file

Reply:
thanks - just committed this